src/App.jsx

```diff
@@ -50,7 +51,16 @@ export function App() {
 		<div className="bg-main-light/[0.4]">
 			<Router>
 				<Routes>
-					<Route path="/" element={<Layout />}>
+					<Route
+						path="/"
+						element={
+							<Layout
+								data={data}
+								listToken={listToken}
+								setAllDetailsOpen={setAllDetailsOpen}
```

Comment:
@neshacascia adding the prop for current state

```suggestion
								setAllDetailsOpen={setAllDetailsOpen}
								allDetailsOpen={allDetailsOpen}
```